src/functionalTest/resources/features/F-004/S-045.td.json

```diff
@@ -0,0 +1,39 @@
+{
+  "_guid_": "S-045",
+  "_extends_": "F-004_Test_Base_data",
+  "title": "must get an error response for a non-existing secutiry classification",
+  "specs": [
+    "contains a non-existing secutiry classification"
+  ],
+  "request": {
+    "headers": {
+      "caseTypeId": "BEFTA_CASETYPE_2_2",
+      "jurisdictionId": "BEFTA_JURISDICTION_2"
+    },
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "classification",
+          "value": "NonExisting"
+        },
+        {
+          "key": "roles",
+          "value": "caseworker"
+        },
+        {
+          "key": "files",
+          "filePath": "SampleImage.png"
+        }
+      ]
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "body" : {
+      "message" : "Exception while uploading the documents :java.lang.IllegalArgumentException: The input parameter does not complies with the required pattern",
+      "path" : "/cases/documents"
```

Comment:
404 response like this?
Seems a wrong message.
And, even if it was a correct one, a message like above containing an Exception name is not good to return to the client.

Reply:
the response changed to 400

Reply:
Jira for the same i.e. AM-648